package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "venmic",
```

if you would like to push this package to npm, it would probably be best to add it to our org. i dont exactly know how npm orgs work but i'll just invite you tmrw (or whenever)
```suggestion
  "name": "@vencord/venmic",
```